xc/xenops_server_xen.ml

```diff
@@ -1429,10 +1426,7 @@ module VM = struct
                                VmExtra.build_info = Some build_info;
                                ty = Some vm.ty;
                              } in
-            Some {
-              VmExtra.persistent = persistent;
-              VmExtra.non_persistent = d.VmExtra.non_persistent;
-            }
+            Some {VmExtra.persistent = persistent}
```

Comment:
I think you can do `Some VmExtra.{persistent} here

Reply:
Fixed.